operator/src/main/resources/scripts/wdt_mii_filter.py

```diff
@@ -0,0 +1,546 @@
+# Copyright (c) 2018, 2021, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+# ------------
+# Description:
+# ------------
+#  This is a WDT filter for handling WLS configuration overrides.
```

Comment:
(1) Minor comment: This is a model-in-image WDT filter for overriding WLS configuration, it replaces 'situational configuration overrides'.

(2) Less minor comment:  Please expand the description to describe (at a relatively high level) this script's command line, env var, and file inputs, plus to describe its expected output in the success and error cases...

Reply:
Thanks for addressing (1), can you take a look at (2)? The idea is to doc how the script fits into the flow for anyone that comes along later that needs to make a modification...

Reply:
Nice! Thanks!